maskbench/maskbench/outlines_engine.py

```diff
@@ -29,10 +31,10 @@ def reset(self):
         self.guide  = Guide(self.index)
 
     def compute_mask(self):
-        self.res = self.guide.get_tokens()
+        fill_next_token_bitmask(self.guide, self.mask)
 
     def commit_token(self, t: int) -> bool:
-        ok = t in self.res
+        ok = True
```

Comment:
Don't you need to check check if token is in mask? It seem advance() will throw if givan in invalid token?